### PR TITLE
feat(monitoring-connector) add documentation for Datacore new plugin …

### PR DESCRIPTION
**Pull request by Evan-Adam**
## Description

Documentation of new Datacore plugin pack

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [X] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="HW-Storage-Datacore-Sansymphony-RESTAPI-custom" label="HW-Storage-Datacore-Sansymphony-RESTAPI-custom">
+
+| Alias           | Modèle de service                                             | Description                                          |
+|:----------------|:--------------------------------------------------------------|:-----------------------------------------------------|
+| Alerts          | HW-Storage-Datacore-Sansymphony-Alerts-RESTAPI-custom         | Contrôle les alertes                                 |
+| Monitors Status | HW-Storage-Datacore-Sansymphony-Status-Monitor-RESTAPI-custom | Contrôle le status des éléments filtré en paramètres |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:25]:
```suggestion
| Monitors Status | HW-Storage-Datacore-Sansymphony-Status-Monitor-RESTAPI-custom | Contrôle le statut des éléments filtrés en paramètres |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+</TabItem>
+</Tabs>
+
+### Règles de découverte
+
+#### Découverte de services
+
+| Nom de la règle                   | Description                                                                                     |
+|:----------------------------------|:------------------------------------------------------------------------------------------------|
+| Datacore-Sansymphony-Restapi-Pool | découvre les identifiant de pools disponible sur le serveur Datacore Sansymphony via l'api Rest |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:49]:
```suggestion
| Datacore-Sansymphony-Restapi-Pool | Découvre les identifiants de pools disponibles sur le serveur Datacore Sansymphony via l'API Rest |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 3]
```diff
@@ -0,0 +1,405 @@
...
+| Métrique                           | Unité |
+|:-----------------------------------|:------|
+| datacore.pool.oversubscribed.bytes | bytes |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+*Specify prerequisites that are relevant. You may want to just provide a link\n\
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:87]: sample text

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 4]
```diff
@@ -0,0 +1,405 @@
...
+4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires.
+
+| Macro           | Description                                                                                                                                        | Valeur par défaut | Obligatoire |
+|:----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| APIUSERNAME     | Username to access the endpoint.                                                                                                                   |                   | X           |
+| APIPASSWORD     | Password to access the endpoint.                                                                                                                   |                   | X           |
+| APIPROTO        | HTTP protocol, either http or https.                                                                                                               | https             |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:190]:
```suggestion
| APIPROTO        | Either HTTP or HTTPS.                                                                                                               | https             |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 5]
```diff
@@ -0,0 +1,405 @@
...
+2. Renseignez les macros désirées (par exemple, ajustez les seuils d'alerte). Les macros indiquées ci-dessous comme requises (**Obligatoire**) doivent être renseignées.
+
+<Tabs groupId="sync">
+<TabItem value="Alerts" label="Alerts">
+
+| Macro           | Description                                                                                                                                                    | Valeur par défaut | Obligatoire |
+|:----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| MAXALERTAGE     | filter alerts to check those newer than this parameter (s)                                                                                                     | 86400             |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:206]:
```suggestion
| MAXALERTAGE     | Filter alerts to check those newer than this parameter (s)                                                                                                     | 86400             |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 6]
```diff
@@ -0,0 +1,405 @@
...
+| FILTERSERVER    | Define which devices should be monitored based on the MachineName. This option will be treated as a regular expression. By default all machine will be checked |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:207]:
```suggestion
| FILTERSERVER    | Define which devices should be monitored based on the MachineName. This option will be treated as a regular expression. By default all machines will be checked |                   |             |
```

Comment (cg-tw) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:207]: Is MachineName a parameter?

File: pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 7]
```diff
@@ -0,0 +1,404 @@
...
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="HW-Storage-Datacore-Sansymphony-RESTAPI-custom" label="HW-Storage-Datacore-Sansymphony-RESTAPI-custom">
+
+| Service Alias   | Service Template                                              | Service Description                   |
+|:----------------|:--------------------------------------------------------------|:--------------------------------------|
+| Alerts          | HW-Storage-Datacore-Sansymphony-Alerts-RESTAPI-custom         | Check alerts                          |
+| Monitors Status | HW-Storage-Datacore-Sansymphony-Status-Monitor-RESTAPI-custom | Check elements filtered by parameters |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:24]: unclear

File: pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 8]
```diff
@@ -0,0 +1,404 @@
...
+</TabItem>
+</Tabs>
+
+### Discovery rules
+
+#### Service discovery
+
+| Rule name                         | Description                                                |
+|:----------------------------------|:-----------------------------------------------------------|
+| Datacore-Sansymphony-Restapi-Pool | discover pools id from your Datacore Sansymphony rest api. |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:48]:
```suggestion
| Datacore-Sansymphony-Restapi-Pool | Discover the IDs of the pools from your Datacore Sansymphony Rest API. |
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 9]
```diff
@@ -0,0 +1,404 @@
...
+
+| Metric name                        | Unit  |
+|:-----------------------------------|:------|
+| datacore.pool.oversubscribed.bytes | bytes |
+
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:85]: sample text

File: pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 10]
```diff
@@ -0,0 +1,404 @@
...
+2. Fill in the macros you want (e.g. to change the thresholds for the alerts). Some macros are mandatory (see the table below).
+
+<Tabs groupId="sync">
+<TabItem value="Alerts" label="Alerts">
+
+| Macro           | Description                                                                                                                                                    | Default value     | Mandatory   |
+|:----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| MAXALERTAGE     | filter alerts to check those newer than this parameter (s)                                                                                                     | 86400             |             |
+| FILTERSERVER    | Define which devices sh ould be monitored based on the MachineName. This option willbe treated as a regular expression. By default all machine will be checked |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:208]:
```suggestion
| FILTERSERVER    | Define which devices should be monitored based on the MachineName. This option will be treated as a regular expression. By default all machines will be checked. |                   |             |
```

Comment (cg-tw) [reply pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:208]: same comment as in FR

File: pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 11]
```diff
@@ -0,0 +1,404 @@
...
+| WARNINGWARNING  | thresholds                                                                                                                                                      |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:209]: Capital T for all instances of "threshold" (and it should be threshold in the singular, shouldn't it?)

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 12]
```diff
@@ -0,0 +1,402 @@
...
+| Metric name                        | Unit  |
+|:-----------------------------------|:------|
+| datacore.pool.oversubscribed.bytes | bytes |
+
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
+Configure Datacore Rest Api : https://docs.datacore.com/RESTSupport-WebHelp/RESTSupport-WebHelp/Installation_Instructions.htm
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:86]:
```suggestion
To be able to monitor Datacore SANsymphony, you must configure an [access to the REST API](https://docs.datacore.com/RESTSupport-WebHelp/RESTSupport-WebHelp/Installation_Instructions.htm).
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 13]
```diff
@@ -0,0 +1,403 @@
...
+| Métrique                           | Unité |
+|:-----------------------------------|:------|
+| datacore.pool.oversubscribed.bytes | bytes |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+Configurer l'Api Datacore Rest : https://docs.datacore.com/RESTSupport-WebHelp/RESTSupport-WebHelp/Installation_Instructions.htm
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:87]:
```suggestion
Pour pouvoir superviser Datacore SANsymphony, il vous faut [configurer l'API REST Datacore](https://docs.datacore.com/RESTSupport-WebHelp/RESTSupport-WebHelp/Installation_Instructions.htm).
```

**Review by omercier — APPROVED** [review]

File: pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 14]
```diff
@@ -0,0 +1,402 @@
...
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="HW-Storage-Datacore-Sansymphony-RESTAPI-custom" label="HW-Storage-Datacore-Sansymphony-RESTAPI-custom">
+
+| Service Alias   | Service Template                                              | Service Description                                     |
+|:----------------|:--------------------------------------------------------------|:--------------------------------------------------------|
+| Alerts          | HW-Storage-Datacore-Sansymphony-Alerts-RESTAPI-custom         | Check alerts                                            |
+| Monitors Status | HW-Storage-Datacore-Sansymphony-Status-Monitor-RESTAPI-custom | Check Datacore "monitors" object filtered by parameters |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:24]:
```suggestion
| Monitors-Status | HW-Storage-Datacore-Sansymphony-Status-Monitor-RESTAPI-custom | Monitor Datacore "monitors" object. |
```

File: pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md [hunk 15]
```diff
@@ -0,0 +1,402 @@
...
+
+| Metric name                   | Unit  |
+|:------------------------------|:------|
+| datacore.event.error.count    | count |
+| datacore.alerts.warning.count | count |
+| datacore.alerts.info.count    | count |
+| datacore.alerts.trace.count   | count |
+
+</TabItem>
+<TabItem value="Monitors Status" label="Monitors Status">
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md:68]:
```suggestion
<TabItem value="Monitors-Status" label="Monitors-Status">
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**